k8s/cmd/commands/inject/inject_argo_workflow.go

```diff
@@ -0,0 +1,666 @@
+	  - dag:
+	      tasks:
+	      - arguments:
+	          parameters:
+	          - name: functions
+	            value: prepare-data.py
+	          - name: vineyard-objects-name
+	            value: '{{tasks.vineyard-objects.outputs.parameters.vineyard-objects-name}}'
+	        dependencies:
+	        - vineyard-objects
+	        name: prepare-data
+	        template: MLops
+	      - arguments:
+	          parameters:
+	          - name: functions
+	            value: preprocess-data.py
+	          - name: vineyard-objects-name
+	            value: '{{tasks.vineyard-objects.outputs.parameters.vineyard-objects-name}}'
+	        dependencies:
+	        - prepare-data
+	        - vineyard-objects
+	        name: preprocess-data
+	        template: MLops
+	      - arguments:
+	          parameters:
+	          - name: functions
+	            value: train-data.py
+	          - name: vineyard-objects-name
+	            value: '{{tasks.vineyard-objects.outputs.parameters.vineyard-objects-name}}'
+	        dependencies:
+	        - preprocess-data
+	        - vineyard-objects
+	        name: train-data
+	        template: MLops
+	      - arguments:
+	          parameters:
+	          - name: functions
+	            value: test-data.py
+	          - name: vineyard-objects-name
+	            value: '{{tasks.vineyard-objects.outputs.parameters.vineyard-objects-name}}'
+	        dependencies:
+	        - train-data
+	        - vineyard-objects
+	        name: test-data
+	        template: MLops
+	      - name: vineyard-objects
+	        template: vineyard-objects
+	    name: dag
+	  - name: vineyard-objects
+	    outputs:
+	      parameters:
+	      - name: vineyard-objects-name
+	        valueFrom:
+	          jsonPath: '{.metadata.name}'
+	    resource:
+	      action: create
+	      manifest: |-
+	        apiVersion: v1
+	        kind: PersistentVolumeClaim
+	        metadata:
+	          name: '{{workflow.name}}-vineyard-objects-pvc'
+	        spec:
+	          accessModes:
+	          - ReadWriteMany
+	          resources:
+	            requests:
+	              storage: 1Mi
+	          storageClassName: vineyard-system.vineyardd-sample.csi
+	      setOwnerReference: true` +
+		"\n```")
+
+	injectArgoWorkflowExample = util.Examples(`
+	# Inject the vineyard volumes into the argo workflow
+	$ vineyardctl inject argo-workflow -f workflow.yaml \
+	     --templates="preprocess-data,train-data" \
+	     --vineyard-cluster="vineyard-system/vineyardd-sample" \
+	     --mount-path="/vineyard/data" \
+	     --dag="dag" \
+	     --tasks="preprocess-data,train-data"
+	
+	# Suppose you only have a single template in the workflow
+	# you could set only one template name in the --templates flag
+	$ vineyardctl inject argo-workflow -f workflow.yaml \
+         --templates="mlops" \
+		 --vineyard-cluster="vineyard-system/vineyardd-sample" \
+		 --mount-path="/vineyard/data" \
+		 --dag="dag" \
+		 --tasks="preprocess-data,test-data"
+	`)
```

Comment:
Can you use a simpler demo case in docstring? for example a workflow with only 2 step.

Reply:
Ok.